docs/Homebrew-on-Linux.md

```diff
@@ -5,7 +5,10 @@ redirect_from:
   - /Linux
   - /Linuxbrew
 ---
+
+<p style="text-align:center">
```

Comment:
I think at the point we're adding this to the Markdown it means we should instead add/modify CSS on brew.sh as @jonchang suggests. Shout if you need help with that.

Reply:
I've opened PR https://github.com/Homebrew/brew.sh/pull/342 to modify the style sheet.